R/pkg/R/DataFrame.R

```diff
@@ -1384,7 +1384,7 @@ setMethod("saveAsTable",
                                     "org.apache.spark.sql.parquet")
             }
             allModes <- c("append", "overwrite", "error", "ignore")
-            if (!(mode %in% allModes)) {
+            if (! (mode %in% allModes)) {
```

Comment:
I sent a PR to add an exclusion for an exclamation before a left parenthesis to the lintr community. If the PR will be merged, then I'll will modify this PR.
https://github.com/jimhester/lintr/pull/109